app/utils.ts

```diff
@@ -292,9 +293,10 @@ export function getMessageImages(message: RequestMessage): string[] {
 }
 
 export function isVisionModel(model: string) {
+  const accessStore = useAccessStore.getState();
   return (
     // model.startsWith("gpt-4-vision") ||
     // model.startsWith("gemini-pro-vision") ||
-    model.includes("vision")
+    model.includes("vision") || accessStore.customVisionModels.includes(model)
```

Comment:
so bad using `||` 
this a4c54cae60820a8a026481b91a11a9e7a842699c a better way

Reply:
It's more difficult to understand the process of constructing a list from customVisionModels:string and the "vision" string.
The approach introduced in this commit is much easier to comprehend.

Reply:
> It's more difficult to understand the process of constructing a list from customVisionModels:string and the "vision" string. The approach introduced in this commit is much easier to comprehend.

no, that not easier, and so bad because of complex logic using `||`

Reply:
I understand your suggestions and find your insights very valuable. Based on your feedback, I have made the following changes to the code:
[ef02ef2b43a3c2ef2150b375549bcad453bfd23c](https://github.com/ChatGPTNextWeb/ChatGPT-Next-Web/pull/4310/commits/ef02ef2b43a3c2ef2150b375549bcad453bfd23c)
Thank you very much for taking the time to review my code and provide valuable feedback. If it's convenient for you, could you please review my changes once more? If there's anything else that needs improvement, please let me know.

Thank you again for your help and support!

Reply:
> I understand your suggestions and find your insights very valuable. Based on your feedback, I have made the following changes to the code: [ef02ef2b43a3c2ef2150b375549bcad453bfd23c](https://github.com/ChatGPTNextWeb/ChatGPT-Next-Web/pull/4310/commits/ef02ef2b43a3c2ef2150b375549bcad453bfd23c) Thank you very much for taking the time to review my code and provide valuable feedback. If it's convenient for you, could you please review my changes once more? If there's anything else that needs improvement, please let me know.
> 
> Thank you again for your help and support!

np, latest changed it more easy to maintain, and more like reusable instead of complexity that using `||`